### ISSUE TITLE
How can I fine-tune the model in chinese question gereration(number 9)

### ISSUE THREAD

**Issue by i-want-steal-sixsix** [yaoxiaoyuan/mimix#5]
I am new in the period and I found your project is quite interesting.I am wondering if I could fine-tine your model in number ix (I got your model quite a long time ago and at that time you didnt ask for qualification).If I could ,what kind of dataset should I prepare and how can I do it(maybe there is script in your project?).Thank you for your help!

**Comment by yaoxiaoyuan** [comment]
I’m very happy this repo can help someone interested in Chinese text generation.  Currently I have not time to complete a well-written documentation for model fine-tuning. Basically, you need to
(1) Prepare a  fine-tune config file. The config file is like model config with several sections, which contains the learning rate, dataset path, etc.  You can find args need to be specified in single trainer file.  
(2) Prepare your dataset. The dataset content is just follow your task input and output. Each line of the dataset file is a json string.
                                                                {"src":input, "trg":output} 
                                                                {"src":input, "trg":output} 
                                                                ...
(3) Run the code. python run_train.py --model_conf model_conf_path --train_conf train_conf_path.

**Comment by i-want-steal-sixsix** [comment]
thank you!
